src/main/resources/application.properties

```
@@ -19,11 +19,11 @@ logging.level.org.hibernate.orm.jdbc.bind=TRACE
 #logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
 #logging.level.org.hibernate.type=trace
 
-## NÃO MODIFICAR ISSO
+## NÃO MODIFICAR ISSO
 spring.devtools.livereload.enabled=false
 spring.devtools.restart.trigger-file=.reloadTriggerIfRestartIsFalse
 
-spring.mail.host=live.smtp.mailtrap.io
+spring.mail.host=sandbox.smtp.mailtrap.io
```

Retornar para live.smtp.mailtrap.io
```suggestion
spring.mail.host=live.smtp.mailtrap.io
```